csvcut.py

```diff
@@ -0,0 +1,103 @@
+from argparse import ArgumentParser
+import sys
+
+
+DESCRIPTION = 'csvpp - prints cvs file in human-readable format'
+EXAMPLES = 'example: cat file.txt | csvpp -f | less -SR'
+
+
+def write_row(index ,row, output_stream):
+    """
+    Write_row modifeis the rows, leaving only the column desired
+    :param index: the indeces of columns that need to remain in the file
+    :param row: row represented as a list of columns
+    :param output_stream: a stream of modified row
+
+    """
+    output_line = ''
+    for i, column in enumerate(row):
+        if(i in index):
```

Comment:
It's better not to indent main code line. I mean, it's better to write
if i not in index:
    continue
# YOUR MAIN CODE HERE